src/build/render/features/index.js

```diff
@@ -0,0 +1,26 @@
+/* Exposes an array of async feature functions, which when given a render context
+ * return an object containing render outputs:
+ *
+ * - keywords: search-boosted array of terms
+ * - searchText: plaintext string to become searchable for this page
+ * - html: added to the body of the article wrapper
+ * - headings: array of {id, title, level} used to form table of contents
+ * - metaTitle: title content of metabox
+ * - metaSections: array of {cssClass, body} to appear as metabox sections
+ * - img: image path for metabox
+ * - imgCaption: caption for metabox
+ * - thanks: added to the credits section at end of page, {name: [forText, ...], ...}
+ *
+ * The order of features determines order on the page.
+ */
+module.exports = [
+  require("./alerts"),
+  require("./metaboxBase"),
+  require("./workflows"),
+  require("./markdownFile"),
+  require("./survey/results"),
+  require("./tagIndex"),
+  require("./tag"),
+  require("./thanks"),
+  require("./thanksIndex"),
+];
```

Comment:
This is pretty key to understanding the new render features system. The render context (containing the current page metadata, language, and APIs like the page index and tag data) is passed through a series of functions which return what they want added to the resulting page and search index. The comment above details what's supported.